### ISSUE TITLE
python in cmd Ctrl+C problem

### ISSUE THREAD

**Issue by EECSGEEK** [lukesampson/scoop#2339]
## python install from scoop (Ctrl+C)KeyboardInterrupt problem in cmd
![py_err.png](https://i.loli.net/2018/06/28/5b345319160fe.png)

### process:
1. open cmd.exe 
2. enter python in cmd
3. in python REPL/shell 【press】【Ctrl+C】
4. now enter a string, will eval by cmd
5. loop with: cmd command -> python command -> cmd command

## python install from python.org don't have (Ctrl+C)KeyboardInterrupt problem
![py_org.png](https://i.loli.net/2018/06/28/5b3456898318e.png)

---
OS: 10.0.17134 64bit
Python: 3.7

**Comment by leonyu** [comment]
This is most likely an issue with the shim, which Scoop creates to deal with common PowerShell/CMD/Windows incompatibilities. Of course the shims are not perfect. 

Try invoke `python.exe` in the directory of `scoop prefix vim` directly,

**Comment by rasa** [comment]
This issue should be fixed in #3998, which was merged into master. To use, type:
```
scoop config shim kiennq
scoop reset *
```
or
```
scoop config shim 71
scoop reset *
```
Eventually, one of those shims (probably kiennq) we be made the default. 

If this issue is still a problem after trying both solutions above, please reopen this issue. Thanks!
